.github/ISSUE_TEMPLATE/3.demo.yml

```diff
@@ -0,0 +1,41 @@
+name: Demo crashed
+description: A demo has crashed
+labels: [docs, 'status: needs triage']
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please take a moment to look at the [existing issues](https://github.com/mui-org/material-ui/issues) to see whether this has already been reported.
```

Comment:
I think making this step required would make most of the people give up on creating the issue. These demos are mainly for us to know if a demo is crashing. I would rather close a few similar issues created in a close interval, rather than miss the report that a demo is crashing.

Reply:
Ok, I see. I don't think this PR will add that much value to the existing solution without making this step required. Did we already consider using something like [sentry.io](https://sentry.io/) for this purpose?

Reply:
Not as far as I am aware of. Does it automatically report crashes like this one?

Reply:
It'll capture and collect exceptions and allows you to aggregate and analyze in their UI